packages/prisma/schema.prisma

```diff
@@ -1091,6 +1091,7 @@ model WorkflowStep {
   sender                    String?
   numberVerificationPending Boolean            @default(true)
   includeCalendarEvent      Boolean            @default(false)
+  safe                      Boolean            @default(false)
```

Comment:
Ok. Hear me out. Why not `markedAsSafeAt` or `verifiedAt`. Timestamp can give us more context on when this became safe instead of a simple true false.

Reply:
This is a great idea, will make that change